kvfifo.h

```diff
@@ -129,13 +144,19 @@ class kvfifo {
   // Dowolna operacja modyfikująca kolejkę może unieważnić zwrócone referencje.
   // Jeśli kolejka jest pusta, to podnosi wyjątek std::invalid_argument.
   // Złożoność O(1).
-  std::pair<K const &, V &> front() { return items.front().as_pair(); }
+  std::pair<K const &, V &> front() {
```

Comment:
Tutaj trzeba coś zrobić, bo w obecnej implementacji w kvfifo_example.cc:20 nie wykonuje się pełna kopia, a chyba powinna się wykonać, bo zwróciliśmy wcześniej referncję w kvfifo_example.cc:18

Reply:
Możemy sobie zapisać w jakimś prywatnym polu, że zwróciliśmy nie-const referencję i jeśli wykonujemy kopię/przypisanie z innego obiektu to sprawdzić czy istnieje jakaś nie-const referencja, bo jeśli istnieje, to musimy zrobić kopię.